File: Assignment_6/Question4.c
#include<stdio.h>

int NumMulti(int iNo1, int iNo2, int iNo3)
{
    if(iNo1 == 0)
    {
        return iNo1 = 1;
    }
    if(iNo2 == 0)
    {
        return iNo2 = 1;
    }
    if(iNo3 == 0)
    {
        return iNo3 = 1;
    }
    
    int iMulti = iNo1 * iNo2 * iNo3;
    return iMulti;
}

int main()
{
    int iValue1 = 0;
    int iValue2 = 0;
    int iValue3 = 0;

    int iResult = 0;

    printf("Enter the first number : \n");
    scanf("%d", &iValue1);

    printf("Enter the second number : \n");
    scanf("%d", &iValue2);

    printf("Enter the third number : \n");
    scanf("%d", &iValue3);

    iResult = NumMulti(iValue1,iValue2,iValue3);

    printf("The multiplication is :%d\n", iResult);

    return 0;
}

File: Assignment__17/Question4.java
//Write a program to find the minimum of three numbers.

import java.util.Scanner;

class logic
{
    int DisplayMinimum(int iNo1, int iNo2, int iNo3)
    {
        if(iNo1 < iNo2 && iNo1 < iNo3)
        {
            return iNo1;
        }
        else if(iNo2 < iNo1 && iNo2 < iNo3)
        {
            return iNo2;
        }
        else
        {
            return iNo3;
        }
    }
}

class Question4
{
    public static void main(String A[])
    {
        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter the first number : ");
        int iValue1 = sobj.nextInt();

        System.out.println("Enter the second number : ");
        int iValue2 = sobj.nextInt();

        System.out.println("Enter the third number : ");
        int iValue3 = sobj.nextInt();

        int iResult = 0;

        logic lobj = new logic();
        iResult = lobj.DisplayMinimum(iValue1, iValue2, iValue3);

        System.out.println("The smallest number is " + iResult);
        
    }    


} 
